### ISSUE TITLE
Cannot install 1.1rc2 on Windows 7

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#4385]
This is the first time I am trying to install a specific version of Astropy on Windows. Previously, I just used whatever that came with the Anaconda installation.

```
$ python
Python 2.7.11 |Anaconda 2.0.1 (64-bit)| (default, Dec  7 2015, 14:10:42) [MSC v.
1500 64 bit (AMD64)] on win32
$ gcc --version
gcc.exe (GCC) 4.7.0 20111220 (experimental)
Copyright (C) 2011 Free Software Foundation, Inc.
```

I tried:

1. Using `pip`
2. Install directly from source tarball, which was downloaded from PyPi.

I get this:

```
The command line is too long.
error: command 'C:\\Users\\username\\AppData\\Local\\Continuum\\Anaconda\\Scripts\\gcc.bat' failed with exit status 1
```

The offending command:

> C:\Users\lim\AppData\Local\Continuum\Anaconda\Scripts\gcc.bat -DMS_WIN64 -shared -s build\temp.win-amd64-2.7\Release\astropy\_erfa\core.o build\temp.win-amd64-2.7\Release\cextern\erfa\a2af.o build\temp.win-amd64-2.7\Release\cextern\erfa\a2tf.o build\temp.win-amd64-2.7\Release\cextern\erfa\ab.o build\temp.win-amd64-2.7\Release\cextern\erfa\af2a.o build\temp.win-amd64-2.7\Release\cextern\erfa\anp.o build\temp.win-amd64-2.7\Release\cextern\erfa\anpm.o build\temp.win-amd64-2.7\Release\cextern\erfa\apcg.o build\temp.win-amd64-2.7\Release\cextern\erfa\apcg13.o build\temp.win-amd64-2.7\Release\cextern\erfa\apci.o build\temp.win-amd64-2.7\Release\cextern\erfa\apci13.o build\temp.win-amd64-2.7\Release\cextern\erfa\apco.o build\temp.win-amd64-2.7\Release\cextern\erfa\apco13.o build\temp.win-amd64-2.7\Release\cextern\erfa\apcs.o build\temp.win-amd64-2.7\Release\cextern\erfa\apcs13.o build\temp.win-amd64-2.7\Release\cextern\erfa\aper.o build\temp.win-amd64-2.7\Release\cextern\erfa\aper13.o build\temp.win-amd64-2.7\Release\cextern\erfa\apio.o build\temp.win-amd64-2.7\Release\cextern\erfa\apio13.o build\temp.win-amd64-2.7\Release\cextern\erfa\atci13.o build\temp.win-amd64-2.7\Release\cextern\erfa\atciq.o build\temp.win-amd64-2.7\Release\cextern\erfa\atciqn.o build\temp.win-amd64-2.7\Release\cextern\erfa\atciqz.o build\temp.win-amd64-2.7\Release\cextern\erfa\atco13.o build\temp.win-amd64-2.7\Release\cextern\erfa\atic13.o build\temp.win-amd64-2.7\Release\cextern\erfa\aticq.o build\temp.win-amd64-2.7\Release\cextern\erfa\aticqn.o build\temp.win-amd64-2.7\Release\cextern\erfa\atio13.o build\temp.win-amd64-2.7\Release\cextern\erfa\atioq.o build\temp.win-amd64-2.7\Release\cextern\erfa\atoc13.o build\temp.win-amd64-2.7\Release\cextern\erfa\atoi13.o build\temp.win-amd64-2.7\Release\cextern\erfa\atoiq.o build\temp.win-amd64-2.7\Release\cextern\erfa\bi00.o build\temp.win-amd64-2.7\Release\cextern\erfa\bp00.o build\temp.win-amd64-2.7\Release\cextern\erfa\bp06.o build\temp.win-amd64-2.7\Release\cextern\erfa\bpn2xy.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2i00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2i00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2i06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2ibpn.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2ixy.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2ixys.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2s.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2t00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2t00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2t06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2tcio.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2teqx.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2tpe.o build\temp.win-amd64-2.7\Release\cextern\erfa\c2txy.o build\temp.win-amd64-2.7\Release\cextern\erfa\cal2jd.o build\temp.win-amd64-2.7\Release\cextern\erfa\cp.o build\temp.win-amd64-2.7\Release\cextern\erfa\cpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\cr.o build\temp.win-amd64-2.7\Release\cextern\erfa\d2dtf.o build\temp.win-amd64-2.7\Release\cextern\erfa\d2tf.o build\temp.win-amd64-2.7\Release\cextern\erfa\dat.o build\temp.win-amd64-2.7\Release\cextern\erfa\dtdb.o build\temp.win-amd64-2.7\Release\cextern\erfa\dtf2d.o build\temp.win-amd64-2.7\Release\cextern\erfa\ee00.o build\temp.win-amd64-2.7\Release\cextern\erfa\ee00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\ee00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\ee06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\eect00.o build\temp.win-amd64-2.7\Release\cextern\erfa\eform.o build\temp.win-amd64-2.7\Release\cextern\erfa\eo06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\eors.o build\temp.win-amd64-2.7\Release\cextern\erfa\epb.o build\temp.win-amd64-2.7\Release\cextern\erfa\epb2jd.o build\temp.win-amd64-2.7\Release\cextern\erfa\epj.o build\temp.win-amd64-2.7\Release\cextern\erfa\epj2jd.o build\temp.win-amd64-2.7\Release\cextern\erfa\epv00.o build\temp.win-amd64-2.7\Release\cextern\erfa\eqeq94.o build\temp.win-amd64-2.7\Release\cextern\erfa\era00.o build\temp.win-amd64-2.7\Release\cextern\erfa\fad03.o build\temp.win-amd64-2.7\Release\cextern\erfa\fae03.o build\temp.win-amd64-2.7\Release\cextern\erfa\faf03.o build\temp.win-amd64-2.7\Release\cextern\erfa\faju03.o build\temp.win-amd64-2.7\Release\cextern\erfa\fal03.o build\temp.win-amd64-2.7\Release\cextern\erfa\falp03.o build\temp.win-amd64-2.7\Release\cextern\erfa\fama03.o build\temp.win-amd64-2.7\Release\cextern\erfa\fame03.o build\temp.win-amd64-2.7\Release\cextern\erfa\fane03.o build\temp.win-amd64-2.7\Release\cextern\erfa\faom03.o build\temp.win-amd64-2.7\Release\cextern\erfa\fapa03.o build\temp.win-amd64-2.7\Release\cextern\erfa\fasa03.o build\temp.win-amd64-2.7\Release\cextern\erfa\faur03.o build\temp.win-amd64-2.7\Release\cextern\erfa\fave03.o build\temp.win-amd64-2.7\Release\cextern\erfa\fk52h.o build\temp.win-amd64-2.7\Release\cextern\erfa\fk5hip.o build\temp.win-amd64-2.7\Release\cextern\erfa\fk5hz.o build\temp.win-amd64-2.7\Release\cextern\erfa\fw2m.o build\temp.win-amd64-2.7\Release\cextern\erfa\fw2xy.o build\temp.win-amd64-2.7\Release\cextern\erfa\g2icrs.o build\temp.win-amd64-2.7\Release\cextern\erfa\gc2gd.o build\temp.win-amd64-2.7\Release\cextern\erfa\gc2gde.o build\temp.win-amd64-2.7\Release\cextern\erfa\gd2gc.o build\temp.win-amd64-2.7\Release\cextern\erfa\gd2gce.o build\temp.win-amd64-2.7\Release\cextern\erfa\gmst00.o build\temp.win-amd64-2.7\Release\cextern\erfa\gmst06.o build\temp.win-amd64-2.7\Release\cextern\erfa\gmst82.o build\temp.win-amd64-2.7\Release\cextern\erfa\gst00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\gst00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\gst06.o build\temp.win-amd64-2.7\Release\cextern\erfa\gst06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\gst94.o build\temp.win-amd64-2.7\Release\cextern\erfa\h2fk5.o build\temp.win-amd64-2.7\Release\cextern\erfa\hfk5z.o build\temp.win-amd64-2.7\Release\cextern\erfa\icrs2g.o build\temp.win-amd64-2.7\Release\cextern\erfa\ir.o build\temp.win-amd64-2.7\Release\cextern\erfa\jd2cal.o build\temp.win-amd64-2.7\Release\cextern\erfa\jdcalf.o build\temp.win-amd64-2.7\Release\cextern\erfa\ld.o build\temp.win-amd64-2.7\Release\cextern\erfa\ldn.o build\temp.win-amd64-2.7\Release\cextern\erfa\ldsun.o build\temp.win-amd64-2.7\Release\cextern\erfa\num00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\num00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\num06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\numat.o build\temp.win-amd64-2.7\Release\cextern\erfa\nut00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\nut00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\nut06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\nut80.o build\temp.win-amd64-2.7\Release\cextern\erfa\nutm80.o build\temp.win-amd64-2.7\Release\cextern\erfa\obl06.o build\temp.win-amd64-2.7\Release\cextern\erfa\obl80.o build\temp.win-amd64-2.7\Release\cextern\erfa\p06e.o build\temp.win-amd64-2.7\Release\cextern\erfa\p2pv.o build\temp.win-amd64-2.7\Release\cextern\erfa\p2s.o build\temp.win-amd64-2.7\Release\cextern\erfa\pap.o build\temp.win-amd64-2.7\Release\cextern\erfa\pas.o build\temp.win-amd64-2.7\Release\cextern\erfa\pb06.o build\temp.win-amd64-2.7\Release\cextern\erfa\pdp.o build\temp.win-amd64-2.7\Release\cextern\erfa\pfw06.o build\temp.win-amd64-2.7\Release\cextern\erfa\plan94.o build\temp.win-amd64-2.7\Release\cextern\erfa\pm.o build\temp.win-amd64-2.7\Release\cextern\erfa\pmat00.o build\temp.win-amd64-2.7\Release\cextern\erfa\pmat06.o build\temp.win-amd64-2.7\Release\cextern\erfa\pmat76.o build\temp.win-amd64-2.7\Release\cextern\erfa\pmp.o build\temp.win-amd64-2.7\Release\cextern\erfa\pmpx.o build\temp.win-amd64-2.7\Release\cextern\erfa\pmsafe.o build\temp.win-amd64-2.7\Release\cextern\erfa\pn.o build\temp.win-amd64-2.7\Release\cextern\erfa\pn00.o build\temp.win-amd64-2.7\Release\cextern\erfa\pn00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\pn00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\pn06.o build\temp.win-amd64-2.7\Release\cextern\erfa\pn06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\pnm00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\pnm00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\pnm06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\pnm80.o build\temp.win-amd64-2.7\Release\cextern\erfa\pom00.o build\temp.win-amd64-2.7\Release\cextern\erfa\ppp.o build\temp.win-amd64-2.7\Release\cextern\erfa\ppsp.o build\temp.win-amd64-2.7\Release\cextern\erfa\pr00.o build\temp.win-amd64-2.7\Release\cextern\erfa\prec76.o build\temp.win-amd64-2.7\Release\cextern\erfa\pv2p.o build\temp.win-amd64-2.7\Release\cextern\erfa\pv2s.o build\temp.win-amd64-2.7\Release\cextern\erfa\pvdpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\pvm.o build\temp.win-amd64-2.7\Release\cextern\erfa\pvmpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\pvppv.o build\temp.win-amd64-2.7\Release\cextern\erfa\pvstar.o build\temp.win-amd64-2.7\Release\cextern\erfa\pvtob.o build\temp.win-amd64-2.7\Release\cextern\erfa\pvu.o build\temp.win-amd64-2.7\Release\cextern\erfa\pvup.o build\temp.win-amd64-2.7\Release\cextern\erfa\pvxpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\pxp.o build\temp.win-amd64-2.7\Release\cextern\erfa\refco.o build\temp.win-amd64-2.7\Release\cextern\erfa\rm2v.o build\temp.win-amd64-2.7\Release\cextern\erfa\rv2m.o build\temp.win-amd64-2.7\Release\cextern\erfa\rx.o build\temp.win-amd64-2.7\Release\cextern\erfa\rxp.o build\temp.win-amd64-2.7\Release\cextern\erfa\rxpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\rxr.o build\temp.win-amd64-2.7\Release\cextern\erfa\ry.o build\temp.win-amd64-2.7\Release\cextern\erfa\rz.o build\temp.win-amd64-2.7\Release\cextern\erfa\s00.o build\temp.win-amd64-2.7\Release\cextern\erfa\s00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\s00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\s06.o build\temp.win-amd64-2.7\Release\cextern\erfa\s06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\s2c.o build\temp.win-amd64-2.7\Release\cextern\erfa\s2p.o build\temp.win-amd64-2.7\Release\cextern\erfa\s2pv.o build\temp.win-amd64-2.7\Release\cextern\erfa\s2xpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\sepp.o build\temp.win-amd64-2.7\Release\cextern\erfa\seps.o build\temp.win-amd64-2.7\Release\cextern\erfa\sp00.o build\temp.win-amd64-2.7\Release\cextern\erfa\starpm.o build\temp.win-amd64-2.7\Release\cextern\erfa\starpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\sxp.o build\temp.win-amd64-2.7\Release\cextern\erfa\sxpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\taitt.o build\temp.win-amd64-2.7\Release\cextern\erfa\taiut1.o build\temp.win-amd64-2.7\Release\cextern\erfa\taiutc.o build\temp.win-amd64-2.7\Release\cextern\erfa\tcbtdb.o build\temp.win-amd64-2.7\Release\cextern\erfa\tcgtt.o build\temp.win-amd64-2.7\Release\cextern\erfa\tdbtcb.o build\temp.win-amd64-2.7\Release\cextern\erfa\tdbtt.o build\temp.win-amd64-2.7\Release\cextern\erfa\tf2a.o build\temp.win-amd64-2.7\Release\cextern\erfa\tf2d.o build\temp.win-amd64-2.7\Release\cextern\erfa\tr.o build\temp.win-amd64-2.7\Release\cextern\erfa\trxp.o build\temp.win-amd64-2.7\Release\cextern\erfa\trxpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\tttai.o build\temp.win-amd64-2.7\Release\cextern\erfa\tttcg.o build\temp.win-amd64-2.7\Release\cextern\erfa\tttdb.o build\temp.win-amd64-2.7\Release\cextern\erfa\ttut1.o build\temp.win-amd64-2.7\Release\cextern\erfa\ut1tai.o build\temp.win-amd64-2.7\Release\cextern\erfa\ut1tt.o build\temp.win-amd64-2.7\Release\cextern\erfa\ut1utc.o build\temp.win-amd64-2.7\Release\cextern\erfa\utctai.o build\temp.win-amd64-2.7\Release\cextern\erfa\utcut1.o build\temp.win-amd64-2.7\Release\cextern\erfa\xy06.o build\temp.win-amd64-2.7\Release\cextern\erfa\xys00a.o build\temp.win-amd64-2.7\Release\cextern\erfa\xys00b.o build\temp.win-amd64-2.7\Release\cextern\erfa\xys06a.o build\temp.win-amd64-2.7\Release\cextern\erfa\zp.o build\temp.win-amd64-2.7\Release\cextern\erfa\zpv.o build\temp.win-amd64-2.7\Release\cextern\erfa\zr.o build\temp.win-amd64-2.7\Release\astropy\_erfa\_core.def -LC:\Users\lim\AppData\Local\Continuum\Anaconda\libs -LC:\Users\lim\AppData\Local\Continuum\Anaconda\PCbuild\amd64 -LC:\Users\lim\AppData\Local\Continuum\Anaconda\PC\VS9.0\amd64 -lpython27 -lmsvcr90 -o build\lib.win-amd64-2.7\astropy\_erfa\_core.pyd

**Comment by pllim** [comment]
Hmm... Duplicate of #3188?

**Comment by embray** [comment]
@pllim It's like one line long--it just tells you the path to the command that's actually being run for your compiler.

**Comment by pllim** [comment]
This is the current contents of `gcc.bat` (which by the way, is in a hidden directory):

```
@echo off
"%~f0\..\..\MinGW\bin\gcc.exe" %*
```

So, you're saying...

1. After it crashes above, manually modify this `gcc.bat` file and execute it once. Then, change it back.
2. ???
3. Astropy 1.1rc2 installed.

**Comment by embray** [comment]
The problem isn't the contents of the batch script.  The problem is the use of a batch script at all, which limits the length of the command-line arguments passed to it.

Instead the best thing to do is just set up your PATH so that for `gcc` it directly picks up `/Path/to/anaconda/MinGW/bin` instead of going through `/path/to/anaconda/<envname>/scripts`.

**Comment by embray** [comment]
Alternatively, it should work if you don't *activate* any anaconda environments.   Just run `/path/to/anaconda/python setup.py` and it should work *as long as* the compiler is on your `PATH` (as described above)

**Comment by pllim** [comment]
@embray , okay, modifying `PATH` works for Python 2.7. However, `python.exe` "stopped working" during `astropy\io\fits\tests\test_checksum.py`. I don't know if it is just my machine, or something to be concerned about for the release. 

In addition, while installing the RC on Python 3.5 environment in Anaconda, I get the error below, which according to StackOverflow, I need MS Visual Studio? Didn't see this error in Python 2.7. Weird!

```
building 'astropy._compiler' extension
error: Unable to find vcvarsall.bat
```

tl;dr - I give up!!!

**Comment by embray** [comment]
It should be fine if you specify `--compiler=unix` when you're in anaconda.  I'm surprised it doesn't automatically default to unix though, but I haven't tried anaconda on Python 3.5 on Windows yet.

I don't know about the other thing.  You should try clean your build environment or something.

The problems you're having building in Anaconda are disconcerting, but not likely particular to astropy in any way.

**Comment by embray** [comment]
Interesting--I think I have a pretty old version of Anaconda on my Windows machine, but when I try to build astropy it uses msvc by default anyways.  But when I specify `--compiler=unix` the build crashes somewhere in distutils, probably due to something specific to anaconda since I don't get that when I use my own build environment.

**Comment by pllim** [comment]
Yeah, not related directly to `astropy` but it still means I am unable to test the RC in Windows. I think `--compiler` is the answer but unfortunately I don't know what is the right thing to pass into it (I tried "unix" and "gcc"). Thanks anyway. Can't say I didn't try! :ghost:

**Comment by embray** [comment]
"gcc" won't do anything.  The "compiler" flag in distutils confusingly does not refer to a specific executable or even the name of a specific compiler generally.  It more represents broad classes of compilers that are assumed to be similar in their interfaces (where "unix" means anything *like* gcc).  

I'll have to see if I can reproduce any of the same difficulty after I update to a more recent anaconda.  It would be good to have some instructions for this.  (I have no problems at all but I'm not using anaconda).

**Comment by stscieisenhamer** [comment]
Just a note: I am able to get it to install, and run tests. There were failures but there is a larger issue with what version of python is actually running. Once I get that worked out, I'll have tests.

Note: The compiler being used is the suggested one from Microsoft: Visual something for Python.

**Comment by mwcraig** [comment]
Compiler version advice:

+ On python 3.5 you should use [MSVC 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx) to compile. I grabbed the community edition and it has been working for building suprisingly well (by which I mean pain-free).
+ On python 2.7 you should install [MSVC for Python](http://www.microsoft.com/en-us/download/details.aspx?id=44266).
+ On python 3.3 or 3.4 it is a little painful. See: https://github.com/cython/cython/wiki/CythonExtensionsOnWindows

To control which compiler (MS vs gnu) in anaconda install (or don't install) libpython:
+ No `libpython` installed: uses MSVC
+ `libpython` installed: use mingw

I've had much better luck with MSVC than mingw with anaconda.

**Comment by pllim** [comment]
Thanks, @mwcraig !

That would explain why my Python 3.5 conda env didn't pick up whatever it used for Python 2.7. But is it possible to have both versions of MSVC installed at the same time? Or does this mean I have to choose to test only one Python version on Windows.

Just to re-iterate, I did finally get Astropy to install with Python 2.7 after changing the `%PATH%` like @embray said. But it "stopped working" (equivalent to Unix's seg fault?) during `io.fits` test. Since you could run the tests on your Windows 7 machine, this must be just me.

@stscieisenhamer, is your Windows machine a STScI-issued laptop?

**Comment by mwcraig** [comment]
@pllim -- I have all three of the microsoft compiler versions installed. There is some additional magic needed for 64-bit python 2.7 and python 3.x < 3.5 that sets up the environment for the 64-bit SDKs (already included for MSVC for 2.7 but a separate download MSVC 2010 for python 3.3 and 3.4). The magic is in astropy/ci-helpers here: https://github.com/astropy/ci-helpers/blob/master/appveyor/windows_sdk.cmd#L54

An alternative is to launch a cmd shell for the SDK (should be under start menu → compiler version → ...tools, maybe?).

**Comment by pllim** [comment]
Thanks, @mwcraig . Sounds doable but very time consuming to set up. I'll stash this as "future work".

:scream: :scream: :scream:

**Comment by stscieisenhamer** [comment]
@pllim i have a tute-issued win VM. At least now I understand the task ahead.
However, I am actually still stuck on just conda and python versions. Regardless of what conda environment i run under, the python version is always the 'root' environment version. Beyond checking then PATH is correct, I haven't had time to dig deeper into that.

**Comment by mwcraig** [comment]
@stscieisenhamer -- how are you launching conda? If it takes you much more time to track down, try posting on the anaconda google group: https://groups.google.com/a/continuum.io/forum/#!forum/anaconda

**Comment by stscieisenhamer** [comment]
win7 using the Anaconda console. Then doing standard conda things: conda create, activate, etc. `python --version`` always gives the same answer regardless of what environment is activated. However, beyond the 5 minutes of "wow that is an issue?" i have not had a chance to get back to it. Wholly plan on writing up when I do.

**Comment by pllim** [comment]
This really reminds me the importance of keeping my Linux machine...

**Comment by embray** [comment]
FWIW I found a few minor test failures when I built and ran the tests on my (somewhat old?) anaconda with Python 2.7.11 32-bit Windows.  I suspect the "32-bit" part is the main reason for the failures, but I haven't checked yet.  I'll update the wiki page with that info.
